prefect_census/utils.py

```diff
@@ -0,0 +1,20 @@
+"""Utilities for common interactions with the Census API"""
+from typing import Optional
+
+from httpx import HTTPStatusError
+
+
+def extract_user_message(error: HTTPStatusError) -> Optional[str]:
+    """
+    Extract user message from an error response from the Census API.
+
+    Args:
+        error: An HTTPStatusError raised by httpx
+
+    Returns:
+        status from Census API response or None if a status cannot
+        be extraacted.
+    """
+    response_payload = error.response.json()
+    status = response_payload.get("status", {})
+    return status
```

Comment:
I think it's more useful to return the message rather than the status in the event of an error.
```suggestion
    user_message = response_payload.get("message", None)
    return user_message
```